crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -394,7 +401,15 @@ type MoveObject {
 }
 
 type MovePackage {
+	"""
+	Given a module name, attempts to deserialize the module from the package
+	The result is a representation of a module in Move that defines struct types
+	and functions that operate on these types.
```

Comment:
Similar comment to the above -- this comment reads imperatively (e.g. here's how GraphQL implements this query), but it's a declarative API, so we should document what it is, not what it does, something like:

```suggestion
	A representation of the module called `name` in this package, including the
	structs and functions it defines.
```